lms/djangoapps/instructor_task/models.py

```diff
@@ -209,10 +209,12 @@ def from_config(cls, config_name):
                 },
             )
         elif storage_type == 'localfs':
+            location = config['ROOT_PATH'] or getattr(settings, 'EDX_GRADES_STORAGE_MEDIA_ROOT', None)
```

Comment:
```getattr(settings, 'EDX_GRADES_STORAGE_MEDIA_ROOT', None)``` calls twice, lets try to avoid this by creating the new variable.

And condition at L212 could be used directly at L216